groups/infrastructure/alb.tf

```diff
@@ -0,0 +1,29 @@
+module "alb" {
+  source = "git@github.com:companieshouse/terraform-modules//aws/application_load_balancer?ref=1.0.205"
+
+  environment                = var.environment
+  service                    = local.service_name
+  ssl_certificate_arn        = aws_acm_certificate.certificate.arn
+  subnet_ids                 = data.aws_subnets.public.ids
```

Comment:
Should this be private.ids?

Reply:
No this should be public. I think it is quite a confusing setup but this follows the pattern set by other ALBs. There are no publicly accessible ALBs in private subnets.